contracts/javascore/xcall/src/test/java/foundation/icon/xcall/CallServiceTest.java

```diff
@@ -285,6 +285,29 @@ public void executeCall_multiProtocol_rollback() throws Exception {
         verify(xcallSpy).CallExecuted(BigInteger.ONE, 1, "");
     }
 
+    @Test
+    public void executeCall_defaultProtocol_rollback() throws Exception {
```

Comment:
Added a test i meant to include in another branch. Coverers a missing branch in the execute call